legend-engine-xts-data-space/legend-engine-xt-data-space-pure-metamodel/src/main/resources/core_data_space_metamodel/metamodel.pure

```diff
@@ -71,7 +71,7 @@ Class meta::pure::metamodel::dataSpace::DataSpacePackageableElementExecutable ex
 Class meta::pure::metamodel::dataSpace::DataSpaceTemplateExecutable extends meta::pure::metamodel::dataSpace::DataSpaceExecutable
 {
    id: String[1];
-   query: FunctionDefinition<Any>[1];
+   query: Any[1];
```

Comment:
As discussed, lets avoid using the type `Any` here. 

Maybe we can have an abstract template class here, with inline and pointer sub classes, with default implementation going to inline to make it backward compatible.